neural_modelling/src/neuron/implementations/neuron_impl_external_devices.h

```diff
@@ -113,9 +113,9 @@ static synapse_param_t *neuron_synapse_shaping_params;
 //! The number of steps to run per timestep
 static uint n_steps_per_timestep;
 
-#ifndef SOMETIMES_UNUSED
-#define SOMETIMES_UNUSED __attribute__((unused))
-#endif // !SOMETIMES_UNUSED
+//#ifndef SOMETIMES_UNUSED
+//#define SOMETIMES_UNUSED __attribute__((unused))
+//#endif // !SOMETIMES_UNUSED
```

Comment:
Why leave commented out if move elsewhere

Reply:
Deleted this and another instance where it was commented out.